src/Symfony/Component/Intl/Resources/data/regions/en.json

```diff
@@ -43,7 +43,7 @@
         "CF": "Central African Republic",
         "CG": "Congo - Brazzaville",
         "CH": "Switzerland",
-        "CI": "Côte d’Ivoire",
+        "CI": "Ivory Coast",
```

Comment:
maybe need to be updated at source
https://github.com/unicode-org/icu/blob/master/icu4c/source/data/region/en.txt#L80